javascript/javascript1/week2/Ali-Nematollahi-homework/student-manager.js

```diff
@@ -0,0 +1,51 @@
+console.log("============ student-manager ============");
+
+const class07Students = [];
+
+function addStudentToClass(studentName) {
+
+  if(class07Students.length < 6 || studentName === "Queen"){
+
+    if (studentName == ""){
```

Comment:
You can write this a bit shorter
```suggestion
    if (!studentName){
```